src/java/htsjdk/samtools/SAMLineParser.java

```diff
@@ -432,6 +453,10 @@ private RuntimeException reportFatalErrorParsingLine(final String reason) {
         return new SAMFormatException(makeErrorString(reason));
     }
 
+    private RuntimeException reportFatalErrorParsingLine(final String reason, final Throwable throwable) {
```

Comment:
Following the convention at line 460.  It avoids me having to write ```SAMFormatException(makeErrorString(...), e)``` at every call site.